src/source/image_source.js

```diff
@@ -198,7 +198,7 @@ class ImageSource extends Evented implements Source {
 
         if (!this.texture) {
             this.texture = new Texture(context, this.image, gl.RGBA);
-            this.texture.bind(gl.LINEAR, gl.CLAMP_TO_EDGE);
+            this.texture.bind(gl.NEAREST, gl.CLAMP_TO_EDGE);
```

Comment:
this is just temporary to see 'nearest' working for Image sources. See comment in the PR.